### DISCUSSION TITLE
install on new Home assistant

### DISCUSSION THREAD

**Discussion by madjuju** [tolwi/hassio-ecoflow-cloud#213] [Q&A]
Hello

Clean install : home assistant 2024.2.2

Advanced mode ok

Instal HACS OK

restart home assistant ok

go to add custom repo of /
[hassio-ecoflow-cloud](https://github.com/tolwi/hassio-ecoflow-cloud)

OK

Go to add new integration : it is not in the list (cache of the browser cleaned, try with safari and chrome) 

Please help

**Comment by madjuju** [comment]
<img width="503" alt="image" src="https://github.com/tolwi/hassio-ecoflow-cloud/assets/5285199/16d6e2b4-d32a-456b-a627-89826c941e2a">

**Reply by 1h8sn0w** [reply]
Just retry or close this modal. You will see that Integration is successfully added anyway.
